isis/src/qisis/objs/FileTool/FileTool.cpp

```diff
@@ -580,9 +580,9 @@ namespace Isis {
         else if ((ocube->pixelType() != Real) &&
                 (ocube->pixelType() != UnsignedByte) &&
                 (ocube->pixelType() != SignedWord) &&
-                (ocube->pixelType() != UnsignedWord)) &&
-                (cube->pixelType() != Isis::UnsignedInteger) &&
-                (cube->pixelType() != Isis::SignedInteger)) {
```

Comment:
How was this even compiling before!?

Reply:
I added this in with my last PR, but it got fixed immediately after. These changes are already in the branch. Not sure why it thinks it's not.